project-react/src/pages/Tablee.js

```diff
@@ -0,0 +1,62 @@
+import axios from "axios";
+import { useState, useEffect } from "react";
+import "./Tablee.css";
+const Tablee = () => {
+  const [posts, setPosts] = useState([]);
+  const apiEndPoint = "https://api.github.com/users/ariv797/repos";

```

Comment:
add this to constant file , and import It here
name file as : endPointUrls for example

Reply:
I created a constant file called ( useApi ) to use Api.